src/components/interactive/Chip.jsx

```diff
@@ -37,7 +47,7 @@ Chip.propTypes = {
   Link: PropTypes.func,
   isActive: PropTypes.bool,
   subtitle: PropTypes.string,
-  hasClose: PropTypes.bool,
+  onClose: PropTypes.func,
```

Comment:
Let's jsdoc all these prop types (the generated props table will pick up jsdoc descriptions)